src/logic/operate.js

```diff
@@ -0,0 +1,15 @@
+const Big = require('big.js');
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+
+  let result = 0;
+
+  if (operation === '-') {
+    result = one - two;
```

Comment:
Kindly work on the project as much as possible so that you can get full feedback that would be helpful;
with the current code we are having;
`result = one - two`
you don't have to be worried about the calculation sign yourself that why we are using `Big.js`
check out the [documentation](https://github.com/MikeMcl/big.js/)

Example: using switch case would be better;
```
switch (operation) {
    case '-':
      result` = one.minus(two)
      break;
```